src/format/reveal/format-reveal.ts

```diff
@@ -116,6 +116,7 @@ export function revealjsFormat() {
           url:
             "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML-full",
         },
+        [kSlideLevel]: 2,
```

Comment:
I was not sure between defining here or below with `link-citations` in extras. But here seemed the true default place (but there must be a reason why `link-citation: true` is below in extras)

Reply:
Yes, that is definitely the right place (I don't recall why we do `link-citation: true` in extras)